certification/internal/shell/has_prohibited_packages.go

```diff
@@ -79,23 +106,23 @@ func (p *HasNoProhibitedPackagesCheck) Help() certification.HelpText {
 // prohibitedPackageList is a list of packages commonly present in the RHEL contianer images that are not redistributable
 // without proper licensing (i.e. packages that are not under the same availability as those found in UBI).
 // TODO: Confirm these packages are the only packages in immediate scope.
-var prohibitedPackageList = []string{
-	"grub",
-	"grub2",
-	"kernel",
-	"kernel-core",
-	"kernel-debug",
-	"kernel-debug-core",
-	"kernel-debug-modules",
-	"kernel-debug-modules-extra",
-	"kernel-debug-devel",
-	"kernel-devel",
-	"kernel-doc",
-	"kernel-modules",
-	"kernel-modules-extra",
-	"kernel-tools",
-	"kernel-tools-libs",
-	"kmod-kvdo",
-	"kpatch*",
-	"linux-firmware",
+var prohibitedPackageList = map[string]struct{}{
+	"grub":                       {},
+	"grub2":                      {},
+	"kernel":                     {},
+	"kernel-core":                {},
+	"kernel-debug":               {},
+	"kernel-debug-core":          {},
+	"kernel-debug-modules":       {},
+	"kernel-debug-modules-extra": {},
+	"kernel-debug-devel":         {},
+	"kernel-devel":               {},
+	"kernel-doc":                 {},
+	"kernel-modules":             {},
+	"kernel-modules-extra":       {},
+	"kernel-tools":               {},
+	"kernel-tools-libs":          {},
+	"kmod-kvdo":                  {},
+	"kpatch*":                    {},
```

Comment:
This glob likely won't match the package name; we might need to explicitly include `kpatch` packages here.

Reply:
I hadn't even noticed that. Let me think on that one.

Reply:
Ok. I have an ok solution. LMKWYT.

Reply:
Globs should work! Resolving